playground/AspireEventHub/EventHubs.AppHost/eventhubns.module.bicep

```diff
@@ -23,26 +23,25 @@ resource eventHubsNamespace_skb4aVCrD 'Microsoft.EventHub/namespaces@2022-10-01-
     name: sku
   }
   properties: {
-    minimumTlsVersion: '1.2'
```

Comment:
Where is this change coming from? Is 1.2 the default and we don't need to explicitly specify it in the bicep?

Also minor nit - why is it generating an empty `properties` object. Couldn't that just be omitted?

Reply:
1.2 would be the default.

The empty properties field might be necessary to satisfy the RP.

Letting @JoshLove-msft know about the change here.